src/avro_copy.cpp

```diff
@@ -216,6 +216,42 @@ struct JSONSchemaGenerator {
 			} else {
 				D_ASSERT(type.id() == LogicalTypeId::MAP);
 				yyjson_mut_obj_add_strcpy(doc, object, "logicalType", "map");
+				auto &list_child = ListType::GetChildType(type);
+				if (field_id) {
+					auto map_field_id = field_id->GetFieldId();
+					if (map_field_id) {
+						yyjson_mut_obj_add_int(doc, object, "field-id", map_field_id);
```

Comment:
This uses `add_int` instead of `add_uint`, I think `int` should only be used if the field id can be negative, which I don't think is the case